actions/ssh.remote.command.yaml

```diff
@@ -0,0 +1,29 @@
+---
+description: Execute command on a remote host
+enabled: true
+entry_point: src/openstack_actions.py
+name: ssh.remote.command
+parameters:
+  timeout:
+    default: 5400
+  lib_entry_point:
+    default: workflows.ssh_remote_command.ssh_remote_command
+    immutable: true
+    type: string
+  host:
+    description: Host to run action on
+    required: true
+    type: string
+  username:
+    description: Username to authenticate as
+    required: true
+    type: string
+  private_key_path:
```

Comment:
We could add support for password encrypted keys, however setting a password would require manual intervention to enter the password not quite sure how this would fit into a automated workflow

Reply:
That is a very good point I did not think about, nevermind!